tests/integration/test_tracing.py

```diff
@@ -0,0 +1,42 @@
+from mock import patch, Mock
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestTracing(IntegrationTestCase):
+
+    def setUp(self, setting_overrides=None):
+
+        self.begin_subsegment = patch('aws_xray_sdk.core.xray_recorder.begin_subsegment')
+        self.begin_subsegment_mock = self.begin_subsegment.start()
+        self.end_subsegment = patch('aws_xray_sdk.core.xray_recorder.end_subsegment')
+        self.end_subsegment_mock = self.end_subsegment.start()
+        self.XRayMiddleware = patch('aws_xray_sdk.ext.flask.middleware.XRayMiddleware', Mock())
+        self.XRayMiddleware_mock = self.XRayMiddleware.start()
+        self.patch_all = patch('aws_xray_sdk.core.patch_all', Mock())
+        self.patch_all_mock = self.patch_all.start()
```

Comment:
I wonder if we could use the patch decorator to clean these up?

Reply:
yes probably, would look cleaner